src/views/teachers/landing/landing.scss

```diff
@@ -0,0 +1,521 @@
+            }
+
+
+            .masthead-info {
+                display: flex;
+                align-items: center;
+                justify-content: space-between;
+
+                p {
+                    margin: 0;
+                    max-width: $cols6;
+                    text-align: left;
+                    color: $ui-white;
+
+                    a {
+                        border-bottom: 1px solid $ui-white;
+                        color: $ui-white;
+                    }
+                }
+
+                iframe {
+                    border: 2px solid $ui-border;
+                    border-radius: 10px;
+                    width: $cols4;
+                    margin-bottom: $gutter;
+                }
+            }
+
+            .band {
+                $band-color: hsla(360, 100, 100, .15);
+
+                margin-top: 2rem;
+                background-color: $band-color;
+                padding: 1rem 0;
+            }
+
+            .sub-nav {
+                text-align: left;
+
+                li {
+                    margin: 0 .5rem 0 0;
+                }
+            }
+
+        }
+
+        &.faq-banner {
+            margin-bottom: 0;
+            background-color: $ui-gray;
+        }
+    }
+
+    .general-usage {
+        justify-content: space-between;
+
+        p {
+            max-width: $cols4;
+        }
+    }
+
+    .educator-community {
+        margin-bottom: 2rem;
+        justify-content: space-between;
+        align-items: flex-start;
+        text-align: left;
+
+        div {
+            max-width: $cols6;
+        }
+    }
+
+    .guides-and-tutorials {
+        div {
+            text-align: center;
+            max-width: $cols4;
+        }
+    }
+
+    section {
+        margin-bottom: 3rem;
+
+        /*p {
+            max-width: $cols8;
+        }*/
+    }
+
+    #in-practice,
+    #resources {
+        .nav-spacer {
+            display: block;
+            visibility: hidden;
+            margin-top: -50px; // height of  nav bar
+            height: 50px;
+        }
+    }
+
+
+    #in-practice {
+        border-bottom: 1px solid $ui-border;
+        padding-bottom: 2rem;
+
+        .stories {
+            align-items: center;
+            margin: auto;
+            justify-content: space-between;
+            flex-direction: row;
+        }
+
+        .story {
+            box-sizing: border-box;
+            margin: 10px auto;
+            border: 1px solid $ui-border;
+            border-radius: 10px;
+            background-color: $ui-white;
+
+            img {
+                border-top-left-radius: 9px;
+                border-top-right-radius: 9px;
+                width: 100%;
+            }
+
+            .story-info {
+                padding-left: 10px;
+                padding-top: 10px;
+
+                p {
+                    margin: 10px 0;
+                    font-size: .75rem;
+                }
+            }
+        }
+    }
+
+    #resources {
```

Comment:
I think it would be best, if possible, to switch these to class names rather than id's.